exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,4 @@
-const name = " Daniel  ";
-
-console.log(message);
+ const myName = " Daniel  ";
+ let result = myName.trim();
+ let message = `My name is ${myName} and my name is ${result.length} characters long.`;
```

Comment:
you should use the variable result instead of myName because  result has the trim() method applied where as myName variable has no trim() method